src/gpioReader.py

```diff
@@ -48,12 +53,14 @@ def __setup(self):
         ## SETUP BME 280 SENSOR
         __bme280 = adafruit_bme280.Adafruit_BME280_I2C(i2c)
         __lsm9ds1 = adafruit_lsm9ds1.LSM9DS1_I2C(i2c)
+        __adxl375 = adafruit_adxl37x.ADXL375(i2c)
 
     
     ## Public Method to be called externally to grab data from sensors
     def retrieveData(self):
         self.__readBME280()
         self.__readLSM9DS1()
+        self.__readADXL375()
```

Comment:
Just saw you were saying you weren't quite sure how we should return data from this method.

One way we could return the data is just with a hash or similar:

```suggestion
        return {
            'bme': self.__readBME280()
            'lsm': self.__readLSM9DS1()
            'adx': self.__readADXL375()
        }
```

Then where ever this method is used, it would look something like this:

```python
data = some_gpio_reader.retrieveData()

print("The ADX got an acceleration of: ", data['adx'])
```